config/utils/stage-ts.js

```
@@ -79,45 +77,17 @@ function escapeContents(contents) {
 }
 
 function compileSass(file) {
-  let contents = '';
-
-  try {
-    contents = sass.renderSync({
-      file,
-      importer: tildeImporter,
-      outputStyle: 'expanded'
-    }).css;
-  } catch (e) {
-    console.log(e.message);
-  }
-
-  return contents;
+  return sass.renderSync({
```

This function (and the one below) should not swallow rendering failures.